compiler/il/OMRDataTypes_inlines.hpp

```diff
@@ -157,6 +157,12 @@ TR::DataTypes OMR::DataType::createMaskType(TR::DataType elementType, TR::Vector
 {
     TR::DataTypes et = elementType.getDataType();
 
+    if (et == TR::Float) {
+       et = TR::Int32;
+    } else if (et == TR::Double) {
+       et = TR::Int64;
+    }
```

Comment:
You mentioned you're making this change because it is an OpenJ9 convention. I'd like to make sure this kind of change makes sense for OMR, so I'll ask some pointy questions (apologies).

Does the mask need to be integral because it would ultimately get turned into either a bitfield or mapped to some hardware operation?  What is the consequence with keeping it as a float?